project_extra/project.py

```diff
@@ -130,44 +132,47 @@ def action_close(self, cr, uid, ids, context=None):
         # task_id.project_id.type_ids[-1].id
         self.write(cr, uid, [task_id.id], vals, context=context)
 
-        # LY send a email reminder to task responsible and Eric.
```

Comment:
@noahzaozao why have this code been commented out? If there's a good reason to comment it out and to keep it, please add a TODO/FIXME on top of it to explain why we need to keep it and what's left to do about it!